### PR TITLE
Added plugin rebar3_edoc_extensions

### PR DESCRIPTION
**Pull request by vkatsuba**
As discussed with @dumbbell, was created plugin [rebar3_edoc_extensions](https://github.com/vkatsuba/rebar3_edoc_extensions) with same logic for generation of documentation. For generate documentation before publish to hex.pm need to use:
```
rebar3 edoc_extensions
```
This command will generate documentation with the same style what is used before.

### CODE REVIEW

**Comment by dumbbell** [comment]
I submitted a pull request to update the `rebar3_edoc_extensions` plugin to override the `edoc` command. It will also fetch PrismJS and GitHub Actions files automatically.

See vkatsuba/rebar3_edoc_extensions#14.

Based on that, it should be simpler to use and integrate this plugin into Khepri.

**Comment by vkatsuba** [comment]
@dumbbell updated with latest release of `rebar3_edoc_extensions`

File: .gitignore [hunk 1]
```diff
@@ -5,9 +5,6 @@ erl_crash.dump
 /_build
 /doc/*
 !/doc/overview.edoc
-!/doc/README.md
-!/doc/github-markdown.css
-!/doc/prism.css
-!/doc/prism.js
 /mix.lock
 /nonode@nohost/
+rebar3
```
Comment (dumbbell) [inline .gitignore:10]: Could you please remove this change? It is unrelated to the pull request.

**Review by dumbbell — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (vkatsuba) [reply .gitignore]: Done

**Review by dumbbell — APPROVED** [review]

**Pull request merged**